source/blender/editors/space_logic/logic_ops.c

```diff
@@ -829,6 +829,182 @@ static void LOGIC_OT_region_flip(wmOperatorType *ot)
   ot->flag = 0;
 }
 
+/* Custom object operators */
+static int python_class_new_invoke(bContext *C, wmOperator *op, const wmEvent *UNUSED(event))
+{
+  /* Better for user feedback. */
+  return WM_operator_props_dialog_popup(C, op, 15 * UI_UNIT_X);
+}
+
+static int custom_object_register_exec(bContext *C, wmOperator *op)
+{
+  PythonComponent *pycomp;
+  Object *ob = CTX_data_active_object(C);
+  char import[MAX_NAME];
+
+  if (!ob) {
+    return OPERATOR_CANCELLED;
+  }
+
+  RNA_string_get(op->ptr, "class_name", import);
+  pycomp = BKE_custom_object_new(import, op->reports, C);
+
+  if (!pycomp) {
+    return OPERATOR_CANCELLED;
+  }
+
+  ob->custom_object = pycomp;
+
+  WM_event_add_notifier(C, NC_LOGIC, NULL);
+
+  return OPERATOR_FINISHED;
+}
+
+static int custom_obeject_create_exec(bContext *C, wmOperator *op)
```

Comment:
custom_obeject_create_exec should be custom_object_create_exec

Reply:
Fixed it! :)